src/Raven.Server/Documents/DocumentFlags.cs

```diff
@@ -60,7 +60,8 @@ public enum NonPersistentDocumentFlags
         LegacyDeleteMarker = 0x8000,
         ForceRevisionCreation = 0x10000,
         AllowDataAsNull = 0x20000,
-        FromResharding = 0x40000
+        FromResharding = 0x40000,
+        UnarchiveFromPatch = 0x80000,
```

Comment:
UnarchiveFromPatch  -> Unarchive? Since the patching is the only way to unarchive a document?

Reply:
Done